jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/layout/GridList.java

```diff
@@ -3,32 +3,35 @@
 import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.elements.base.UIListBase;
 import com.epam.jdi.light.material.asserts.layout.GridListAssert;
+import com.epam.jdi.light.material.interfaces.base.HasImage;
 
 /**
- * To see an example of Grid List web element please visit
- * https://mui.com/components/grid/
+ * Represents Grid List MUI component on GUI.
+ * <p>
+ * Grid lists display a collection of images in an organized grid.
+ *
+ * @see <a href="https://v3.mui.com/demos/grid-list/#grid-list">Grid List MUI documentation</a>
+ * @see <a href="https://jdi-testing.github.io/jdi-light/material">MUI test page</a>
  */
+public class GridList extends UIListBase<GridListAssert> implements HasImage {
 
-
-public class GridList extends UIListBase<GridListAssert> {
-
-    @Override
-    public GridListAssert is() {
-        return new GridListAssert().set(this);
+    @JDIAction("Check that '{name}' have image")
+    public boolean hasImage(int index) {
+        return get(index).core().find("img").isExist();
     }
 
-    @JDIAction("Is {name} have image")
-    public boolean hasImage(int index) {
-        return get(index).find("img").isExist();
+    @JDIAction("Get '{name}' image alternative name")
+    public String altImgName(int numEl) {
+        return get(numEl).core().find("img").attr("alt");
     }
 
-    @JDIAction("Get {name} image alternative name")
-    public String getAltImgName(int numEl) {
-        return get(numEl).find("img").attr("alt");
+    @JDIAction("Get '{name}' title")
+    public String title(int numEl) {
+        return get(numEl).core().find("div[class*='title']").text().replaceAll("[\\t\\n\\r]+", " ");
```

Comment:
- `numEl` and `index` have the same These words have the same meaning in the context of these methods. It is possible to use the same variable name in all cases.
- I think it's worth adding information about the image index to the action description.
-  Is it really necessary to use `core()` after `get()` here?